pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -19,13 +19,16 @@
   codeorg_homepage_hoc2018_header_line2_mobile: 'What will you create?'
   codeorg_homepage_hoc2018_header_line3: 'Give flight to imagination and birth to innovation. Start with an Hour of Code and your own brand of creativity, whatever it may be.'
   codeorg_homepage_hoc2018_video: 'Watch our video'
+  hourofcode_homepage_hoc2018_header_line1: 'Anyone, anywhere can organize an Hour of Code event. One-hour tutorials in over 45 languages. No experience needed.'
+  hourofcode_homepage_hoc2018_header_line2: 'Ages 4 to 104.'
 
```

Comment:
teeny tiny nit: Does it make sense to put these strings in order: hourofcode_homepage_hoc2018_header_line1, then hourofcode_homepage_hoc2018_header_line2, then codeorg_homepage_hoc2018_header_line3 (currently on line 20)?

Reply:
Aren't they separate sets?

Reply:
Brendan is correct about my intention. The first set is for the code.org homepage and the second send is for the hourofcode.com homepage.